packages/mime-types/package.json

```diff
@@ -1,6 +1,6 @@
 {
   "name": "@shopify/mime-types",
-  "version": "0.0.0",
+  "version": "0.0.1",
```

Comment:
I think the version bump happens when you do a release.

```
Follow the prompts to choose a version for each package. If you are releasing a new package, we encourage you to version it 1.0.0 to start with.
```

https://github.com/Shopify/quilt/blob/master/documentation/guides/release-and-deploy.md

Reply:
Ahh that's good to know thank you